src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -34,17 +34,18 @@ public class Sender : EndpointConfigurationBuilder
         {
             public Sender()
             {
-                var basePath = AppDomain.CurrentDomain.BaseDirectory;
-
-                File.WriteAllLines(Path.Combine(basePath, "DistributingACommand.Receiver.txt"), new[]
-                {
-                    "1:",
-                    "2:"
-                });
+                var filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "routes.xml");
+                File.WriteAllText(filePath, @"<endpoints>
+    <endpoint name=""DistributingACommand.Receiver"">
+        <instance scaleOutDiscriminator=""1""/>
```

Comment:
does `scaleOut` add any value? (we still need to explain to them what a discriminator is?) rename to `discriminator` ?